todo/src/App.js

```diff
@@ -5,9 +5,12 @@ import ListOfTodos from "./components/ListOfTodos";
 import { nanoid } from "nanoid";
 
 function App() {
+  // What is this state for? Could you give it a more descriptive name showing what it means?
   const [todo, setTodo] = useState("");
   const [listOfTodos, setListOfTodos] = useState([]);
+  // This state is never read? Why does it exist?
   const [editAvailable, setEditAvailable] = useState(false);
```

Comment:
@illicitonion  yes, I am using it inside of different components but I just use setEditAvailable, I did not use editAvailable anywhere, is this okay?